stm32-lvgl/.swift-version

```diff
@@ -0,0 +1 @@
+main-snapshot-2025-03-28
```

Comment:
can we move this to the repo root? It would be nice to have all examples on the same version. We should later update the GitHub actions to reference this file.

Reply:
done